firstservice/src/main/java/com/services/firstservice/FirstserviceApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.services.firstservice;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.client.RestTemplate;
+
+@SpringBootApplication
+public class FirstserviceApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(FirstserviceApplication.class, args);
+    }
+
+    @Bean
+    public RestTemplate getRestTemplate() {
+        return new RestTemplate();
+    }
```

Comment:
This bean should be moved to a separate configuration file. I suggest creating `config` folder and inside this folder you should have a class like `AppConfig` where you place your bean.